docker-compose.yml

```
@@ -21,17 +21,8 @@ services:
-      - ./backend/.env   #sh -c "python manage.py wait_for_db && python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
-    environment:
-      DJANGO_SECRET_KEY: 'django-insecure-ihi)mr4kts@_9u3^$9)zegd5sibqmww14d2rv+w%g$eu05k!e8'
-      DJANGO_DEBUG: "True"
-      DJANGO_ALLOWED_HOSTS: "localhost,127.0.0.1"
-      DB_ENGINE: django.db.backends.mysql
-      DB_NAME: learnathon
-      DB_USER: mhpcoder
-      DB_PASSWORD: justkidding12!@
-      DB_HOST: db
-      DB_PORT: "3306"
+      - .env   #sh -c "python manage.py wait_for_db && python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)